recipe/meta.yaml

```
@@ -12,17 +12,18 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
 
 requirements:
   host:
-    - pip >=21.3
-    - python >=3.8
+    - pip
+    - python
     - flit
+    - wheel
   run:
-    - numpy >=1.20
-    - python >=3.8
+    - numpy {{ numpy }}
+    - python
     - scipy >=1.6
```

```suggestion
    - scipy >=1.7
```
For v0.6.0 https://github.com/arviz-devs/xarray-einstats/blob/4d9b8c76284e4e56f1691060b200dd4b83424382/pyproject.toml#L30